src/app/component/quiz/quiz.component.html

```diff
@@ -1,108 +1,145 @@
-<div class="container">
-  <form [formGroup]="quizForm">
-    <div formArrayName="form">
-      <ngb-carousel
-        #carousel
-        *ngIf="question?.length"
-        [showNavigationArrows]="false"
-        [showNavigationIndicators]="false"
-        [interval]="0"
-        [keyboard]="false"
-      >
-        <ng-template
-          ngbSlide
-          *ngFor="let arrayloop of formArray.controls; let i = index"
-        >
-          <ng-container [formGroupName]="i">
-            <div class="p-5">
-              <div class="card card-layout">
-                <div class="card-body card-body-layout">
-                  <div class="quiz">
-                    <div class="data">
-                      {{ "Q-" + (i + 1) }} {{ question[i]["question"] }}
-                    </div>
-                    <div class="timer">
-                      <i class="fa fa-clock timer"
-                        >&#xf017;
-                        {{ formArray.controls.at(i)?.value?.timer }} sec</i
+<form [formGroup]="quizForm">
+  <div formArrayName="form">
+    <div class="container-fluid">
+      <div class="d-flex align-items-center min-vh-100">
+        <div class="container mx-auto">
+          <ngb-carousel
+            #carousel
+            *ngIf="question?.length"
+            [showNavigationArrows]="false"
+            [showNavigationIndicators]="false"
+            [interval]="0"
+            [keyboard]="false"
+          >
+            <ng-template
+              ngbSlide
+              *ngFor="let arrayloop of formArray.controls; let i = index"
+            >
+              <ng-container [formGroupName]="i">
+                <div class="m-3">
+                  <div class="card quiz_card h-auto">
+                    <div
+                      class="card-header header_txt py-0 px-3 d-flex align-items-center justify-content-between position-relative"
+                    >
+                      <div class="header_title user-select-none">
+                        {{ [this.selectedQuizType] }}
```

Comment:
why used like this and never use this inside of html file

Reply:
yes, i changed it.